### ISSUE TITLE
admin/user/:id crashes on direct access

### ISSUE THREAD

**Issue by sgade** [sgade/hgots-node#19]
Commit cb499b0e803c925de3b2b07f81f03f137448585c introduces the admin/user/:userid route.
It works fine, however, if you access it directly the following message comes up:

```
Error while loading route: Error: No model was found for 'id'
```

Anybody any idea how to fix this?

**Comment by cjmoeller** [comment]
Is there anyone out there?

**Comment by sgade** [comment]
**You** seem to be there at least.

**Comment by jhbruhn** [comment]
Finally someone who's receiving my signals. So: I've been sitting here on this Planet. I think you humans call it Alpha Prima. It's pretty warm on here so it might be a star but I'm not quite sure. I've been sending my signals since dozens of years. You usually just put it off as some kind of space noise or whatever you call it, but it's been me trying to communicate in all the languages I am capable of. Now, you FINALLY HEARD ME! YUIS! (that's ronflongish for 'yeah'). So, has anyone seen my water?

**Comment by sgade** [comment]
Fixed in e9329523ffe229183f478ca2a1b2a3e608f1f32e.
